src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -619,25 +928,29 @@
 											DoubleTapped="SelectionCheckbox_DoubleTapped"
 											Opacity="0"
 											Unchecked="ItemSelected_Unchecked" />
+
 									</Grid>
 
+									<!--  Item Name  -->
 									<Grid
 										Width="{Binding ColumnsViewModel.NameColumn.LengthIncludingGridSplitter.Value, ElementName=PageRoot, Mode=OneWay}"
-										Padding="12,0,0,0"
+										Padding="16,0,0,0"
 										HorizontalAlignment="Left"
```

Comment:
Was this on purpose?

Reply:
To bridge the gap between header and content more. But still remains some.

Reply:
This is out of scope for this PR, can you revert?